lib/report/generateReport.ts

```diff
@@ -52,9 +53,18 @@ export interface ErrorDefinition {
   link?: string;
 }
 
+export interface ValidationPassOperationsFormatInner extends OperationMeta {
```

Comment:
the `OperationMeta` is referenced from the [existing code](https://github.com/ms-zhenhua/oav/blob/ed2352f4e9fed02091252140c248f10ca22b9721/lib/swaggerValidator/trafficValidator.ts#L64). `ValidationPassOperationsFormatInner` is similar with [unCoveredOperationsFormatInner](https://github.com/ms-zhenhua/oav/blob/ed2352f4e9fed02091252140c248f10ca22b9721/lib/swaggerValidator/trafficValidator.ts#L68C18-L68C48). Does it make sense to rename it as `passedOperationsFormatInner` ?

Reply:
Frankly I don't think we should keep awkward naming conventions just because it's awkward elsewhere.

The lack of basic comments for complex types describing the WHY of these magic splits is my single biggest frustration of this codebase. "An operationId looks like this, which is why the arbitrary split works" MAKES THE CODE SO MUCH EASIER to walk.